### PR TITLE
Add option to opt-out for hyphenation in Link-/Button-Tile (Issue/2740)

### PR DESCRIPTION
**Pull request by henrijoss**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://opensource.porsche.com/docs/cla)

#### References

- Preview: https://designsystem.porsche.com/issue/2740

#### Scope

Short description of the pull request scope

#### Resolved Issue

Resolves [2740](https://github.com/orgs/porsche-design-system/projects/14/views/1?pane=issue&itemId=36233069)

### CODE REVIEW

File: packages/components/src/components/link-tile-model-signature/link-tile-model-signature-styles.ts [hunk 1]
```diff
@@ -50,13 +50,15 @@ export const getComponentCss = (
     },
     heading: {
       margin: 0,
-      ...textLargeStyle,
+      font: textLargeStyle.font,
```
Comment (stephanschroeter) [inline packages/components/src/components/link-tile-model-signature/link-tile-model-signature-styles.ts:53]: I wouldn't only access `.font`, what happens if we extend the css rules for `textLargeStyles` in the future?

Comment (stephanschroeter) [reply packages/components/src/components/link-tile-model-signature/link-tile-model-signature-styles.ts:53]: And what happend to the previous auto hyphenation styles?

Comment (stephanschroeter) [reply packages/components/src/components/link-tile-model-signature/link-tile-model-signature-styles.ts:53]: Maybe `...textLargeStyle, hypens: 'inherit'` would be more future proof?

File: packages/components/src/components/link-tile/link-tile.examples.md [hunk 2]
```diff
@@ -236,6 +247,21 @@ export default class Code extends Vue {
...
   <img src="${require('@/assets/image-grid.png')}" ${this.imgAttributes} />
 </p-link-tile>`};
+
+  hyphen = 'manual';
+  hyphens = ['auto', 'manual', 'none'];
+  get hyphenMarkup() {
+    return `<p-link-tile
+  href="https://www.porsche.com"
+  label="Some label"
+  description="An extra&shy;ordinarily Porsche"
```
Comment (denyo) [inline packages/components/src/components/link-tile/link-tile.examples.md:257]: Should `&shy;` be kept around for all variants?
What about the cursor/tooltip behavior?

https://github.com/porsche-design-system/porsche-design-system/assets/4380295/6077a618-0d48-4b85-b478-52b0fbe27267

File: packages/components/CHANGELOG.md [hunk 3]
```diff
@@ -18,6 +18,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Theme property supports `auto` for all themeable components, reflecting `prefers-color-scheme` based on OS system
   settings ([#2719](https://github.com/porsche-design-system/porsche-design-system/pull/2719))
+- `hyphens` css attribute can now be overwritten in the `Link Tile`, `Link Tile Model Signature` and `Button Tile`
```
Comment (denyo) [inline packages/components/CHANGELOG.md:21]:
```suggestion
- `hyphens` CSS property can now be overwritten in `Button Tile`, `Link Tile` and `Link Tile Model Signature`
```

File: packages/components/src/components/link-tile/link-tile.examples.md (hunk 2 above)
Comment (henrijoss) [reply packages/components/src/components/link-tile/link-tile.examples.md:257]: I guess it makes sense to keep it in order to show when it has an effect.

**Pull request merged**